test/actions/linkActions.test.js

```diff
@@ -0,0 +1,39 @@
+import test from 'blue-tape'
+import * as actions from '../../src/actions/linkActions'
+
+const CALENDAR = {type: 'course', id: 'MI-RUB', date: '2015-10-11'}
+
+test('calendarUrl()', t => {
+  t.test('calendarUrl() with all parameters', st => {
+    st.plan(1)
```

Comment:
I decided to start with single assertion so I can see the test failing even before writing one. Given the preparation ceremony required for this function, I assume you will need just one assertion for each subtest. I could change it to end(), but I also like the additional check incorporated into plan().